nim/whip/server.nimble

```diff
@@ -0,0 +1,11 @@
+# Package
+
+version = "0.1.0"
+author = "Willyboar"
+description = "Whip Implementation"
+license = "MIT"
+
+# Dependencies
+
+requires "nim >= 1.0.0"
+requires "whip >= 0.0 & < 1.0"
```

Comment:
I prefer `>= 0.1 & < 0.2`

Reply:
There is no version. :(

Reply:
Yeah, but the idea is more to be consistent with the version number display in result ;-)

I mean, in result, I display `X.Y`

Reply:
Done.